astroquery/alma/core.py

```
@@ -18,14 +18,23 @@
 from astropy.utils.console import ProgressBar
 from astropy import units as u
 import astropy.coordinates as coord
-import astropy.io.votable as votable
 
-from ..exceptions import (RemoteServiceError, TableParseError,
-                          InvalidQueryError, LoginError)
+from ..exceptions import (RemoteServiceError, LoginError)
 from ..utils import commons
 from ..utils.process_asyncs import async_to_sync
 from ..query import QueryWithLogin
 from . import conf, auth_urls
+from ..vo import VoImageQuery
+try:
+    import pyvo
+except ImportError:
+    print('Please install pyvo. astropy.vo does not work without it.')
```

this should probably just raise an exception.  Should this also go into `..vo`?